extensions/visual-studio-code-recent-projects/src/index.tsx

```diff
@@ -148,24 +169,8 @@ function LocalItem(props: { entry: EntryLike; uri: string; pinned?: boolean } &
       actions={
         <ActionPanel>
           <ActionPanel.Section>
-            <Action
-              title={`Open in ${build}`}
-              icon="action-icon.png"
-              onAction={() => {
-                if (closeOtherWindows) {
-                  runAppleScriptSync(`
-                    tell application "System Events"
-                      tell process "${build}"
-                        repeat while window 1 exists
-                          click button 1 of window 1
-                        end repeat
-                      end tell
-                    end tell
-                  `);
-                }
-                open(props.uri, bundleIdentifier);
-              }}
-            />
+            <Action title={getTitle()} icon="action-icon.png" onAction={getAction()} />
+            <Action title={getTitle(true)} icon="action-icon.png" onAction={getAction(true)} />
```

Comment:
Currently the Action for the shortcut ⌘ + ⏎ is "Show in Finder". With the new action the order changes and the new action has this shortcut.

Users who have gotten used to the previous action suddenly close all other windows by accident.

Maybe switch the order of the actions or assign fixed shortcuts? What do you think?

Suggestion: 
- new action =>  ⌘ + ⇧ + ⏎
- "Show in Finder" =>  ⌘ + ⏎